camera.go

```diff
@@ -293,8 +319,36 @@ type MouseZoomer struct {
 func (*MouseZoomer) Priority() int          { return 10 }
 func (*MouseZoomer) Remove(ecs.BasicEntity) {}
 
-func (c *MouseZoomer) Update(dt float32) {
+func (c *MouseZoomer) Update(float32) {
 	if Mouse.ScrollY != 0 {
+		// TODO: not sure if we have to use dt here, since we already use the "amount" of zooming
```

Comment:
Done. Decided that we don't need `dt`, because we are already using "delta Y-coordinate".